src/nanovg.c

```diff
@@ -2119,22 +2119,30 @@ void nvgRect(NVGcontext* ctx, float x, float y, float w, float h)
 
 void nvgRoundedRect(NVGcontext* ctx, float x, float y, float w, float h, float r)
 {
-	if (r < 0.1f) {
-		nvgRect(ctx, x,y,w,h);
+	nvgRoundedRectEx(ctx, x, y, w, h, r, r, r, r);
+}
+
+void nvgRoundedRectEx(NVGcontext* ctx, float x, float y, float w, float h, float r1, float r2, float r3, float r4)
+{
+	if(r1 < 0.1f && r2 < 0.1f && r3 < 0.1f && r4 < 0.1f) {
+		nvgRect(ctx, x, y, w, h);
 		return;
 	}
 	else {
```

Comment:
I don't recall changing the formatting, maybe Visual Studio did that to me automatically.